package.json

```diff
@@ -65,6 +65,7 @@
     "@compodoc/compodoc": "^1.0.0-beta.9",
     "@ngtools/webpack": "^7.2.1",
     "@types/jest": "^23.0.0",
+    "@types/googlemaps": "^3.30.16",
```

Comment:
eventually we may want to use an external google maps typing, but for now we are using internal ones, so this dependency is unnecessary